common/djangoapps/student/tests/test_tasks.py

```diff
@@ -0,0 +1,48 @@
+"""
+Tests for the Sending activation email celery tasks
+"""
+
+import mock
+
+from django.test import TestCase
+from django.conf import settings
+from student.tasks import send_activation_email
+
+
+class SendActivationEmailTestCase(TestCase):
+    """
+    Test for send activation email to user
+
+    """
+    @mock.patch('time.sleep', mock.Mock(return_value=None))
+    @mock.patch('student.tasks.log')
+    @mock.patch('django.contrib.auth.models.User')
+    def test_send_email(self, mock_user, mock_log):
+
+        """
+        Test Send activation email exception
+        """
+        from_address = "task_testing@edX.com"
+        mock_user.email_user.side_effect = Exception
+        email_max_attempts = settings.RETRY_ACTIVATION_EMAIL_MAX_ATTEMPTS
+        subject = "Task_test"
```

Comment:
Doesn't seems like we need `subject` and `message` as variables as its only being used a 1 place. 
Also we should use `''` rather than `""`